doc/rst/language/spec/procedures.rst

```diff
@@ -458,11 +458,11 @@ The Out Intent
 
 When ``out`` is specified as the intent, the actual argument is ignored
 when the call is made, but when the function returns, the actual argument
-is assigned to the value of the formal argument.  An implicit conversion
-occurs from the type of the formal to the type of the actual. The actual
-argument must be a valid lvalue. Within the function body, the formal
-argument is initialized to its default value if one is supplied, or to
-its type’s default value otherwise. The formal argument can be modified
+is assigned to the value of the formal argument. The actual argument must
+be a valid lvalue. Within the function body, the formal argument is
+initialized according :ref:`Split_Initialization`.  It will start with
+its default value if one is supplied and can use the type's default value
+if no initialization point is found.  The formal argument can be modified
 within the function.
```

Comment:
In the above:
* The assignment actual->formal should be reversed: `the value of the formal argument is assigned to the actual argument`.
* Where is the right place to specify that an `out` formal can split-initialize the actual?